### ISSUE TITLE
Incorrect endpoints

### ISSUE THREAD

**Issue by shinji257** [netbootxyz/netboot.xyz#1448]
[]: This bug persists when memory is increased beyond 8GB.
N/A

**Describe the bug**
When downloading for local assets I discovered entries in the current release version that are unable to be downloaded due to the endpoints.yml pointing to the wrong file.

https://github.com/netbootxyz/netboot.xyz/blob/515d77d8ca967d373b831eb3334609c2e79add9d/endpoints.yml#L1149
https://github.com/netbootxyz/netboot.xyz/blob/515d77d8ca967d373b831eb3334609c2e79add9d/endpoints.yml#L1165
Both of these should probably be proxmox.iso rather than the full filename.  This was fixed elsewhere btw so that it would download correctly if you still used the default endpoint.

https://github.com/netbootxyz/netboot.xyz/blob/515d77d8ca967d373b831eb3334609c2e79add9d/endpoints.yml#L1587
Actually the `filesystem.squashfs` for this one just doesn't exist on github releases.

https://github.com/netbootxyz/netboot.xyz/blob/515d77d8ca967d373b831eb3334609c2e79add9d/endpoints.yml#L1181
https://github.com/netbootxyz/netboot.xyz/blob/515d77d8ca967d373b831eb3334609c2e79add9d/endpoints.yml#L1193
I'm not sure on these because there are multiple squashfs files showing for the release.  None of them are named `filesystem.squashfs` all by itself.

**TL;DR**
Name mismatch vs local asset download source
proxmox-backup-server	3.1-1-01e6c736/proxmox-backup-server.iso
proxmox-mailgateway	8.1-1-5b1d231d/proxmox-mailgateway.iso	
ubuntu-netboot-22.04-amd64	22.04.4-69164c58/filesystem.squashfs	
ubuntu-netboot-22.04-arm64	22.04.4-6c390eee/filesystem.squashfs	

Just doesn't exist...
ubuntu-netboot-23.10-arm64	23.10-729b6a9e/filesystem.squashfs

**Comment by antonym** [comment]
Closing, think this was resolved a while back.
